pkg/auth/tokenparser.go

```diff
@@ -0,0 +1,83 @@
+package auth
+
+import (
+	"errors"
+	"log"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+// TokenClaims represents access token claims
+type TokenClaims struct {
+	Name          string         `json:"name"`
+	Username      string         `json:"preferred_username"`
+	GivenName     string         `json:"given_name"`
+	FamilyName    string         `json:"family_name"`
+	Email         string         `json:"email"`
+	EmailVerified bool           `json:"email_verified"`
+	Company       string         `json:"company"`
+	SessionState  string         `json:"session_state"`
+	Approved      bool           `json:"approved"`
```

Comment:
It's OSIO legacy staff. Let's remove `Approved` claim.

Reply:
fixed here: 8fff4ce36eadcb70979eebcc6b6d5609c94ecbad